src/i18n.py

```diff
@@ -263,16 +263,17 @@ def __call__(self, untranslated):
         """Main function.
 
         This is the function which is called when a plugin runs _()"""
-        normalizedUntranslated = normalize(untranslated, True)
+        if untranslated.__class__ is InternationalizedString:
```

Comment:
Would `isinstance(untranslated, InternationalizedString)` be more idiomatic?

Reply:
it is, but it's noticeably slower:

```
In [1]: from supybot.i18n import InternationalizedString

In [2]: s1 = "foo bar"

In [3]: s2 = InternationalizedString("foo bar")

In [4]: %timeit s1.__class__ is InternationalizedString
20.7 ns ± 0.043 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [5]: %timeit isinstance(s1, InternationalizedString)
39 ns ± 0.237 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [6]: %timeit s2.__class__ is InternationalizedString
21.4 ns ± 0.0408 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [7]: %timeit isinstance(s2, InternationalizedString)
26.7 ns ± 0.0825 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [8]: %timeit s1.__class__ is InternationalizedString
20.9 ns ± 0.0671 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [9]: %timeit isinstance(s1, InternationalizedString)
38.9 ns ± 0.164 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [10]: %timeit s2.__class__ is InternationalizedString
21.4 ns ± 0.0499 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [11]: %timeit isinstance(s2, InternationalizedString)
26.4 ns ± 0.0881 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```

I've seen non-Limnoria Python code where `isinstance` is a bottleneck; and I wouldn't be surprised if it happened to some plugin doing strange things with strings.